pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-opensteamworks/default.nix

```diff
@@ -2,22 +2,21 @@
 
 stdenv.mkDerivation rec {
   name = "pidgin-opensteamworks-${version}";
-  version = "1.6.1";
+  version = "git-2018-08-02";
```

Comment:
On package naming [the manual](https://nixos.org/nixpkgs/manual/#sec-package-naming) rather suggest `unstable-2018-08-02`

Otherwise you get half the repository when looking for "git" https://nixos.org/nixpkgs/manual/#sec-package-naming :)